tests/naive.py

```diff
@@ -559,17 +602,25 @@ def multi_distance_profile(query_idx, T, m, include=None, discords=False):
     return D_prime_prime
 
 
-def mstump(T, m, excl_zone, include=None, discords=False):
+def mstump(T, m, excl_zone, include=None, discords=False, Ts_subseq_isconstant=None):
+    # Ts_subseq_isconstant is None, or list
     T = T.copy()
 
     d, n = T.shape
     k = n - m + 1
 
+    if Ts_subseq_isconstant is None:
+        Ts_subseq_isconstant = [None] * d
```

Comment:
In ostinato, input is `Ts` (a list of time series) and hence it is reasonable to choose the name `Ts_susbeq_isconstant`. 

Here, in `mstump`, the input `T` is a 2D numpy.ndarray. However, we still need to consider type `list` for the isconstant param. This is because a user may provide a list  of callable objects. Hence, I was wondering if it should be okay to choose the name `Ts_subseq_isconstant` here.

Reply:
No, I would keep the name as `T_subseq_isconstant`. In the case of `ostinato`, the time series are unrelated. In this case, it is clear that the time series, `T`, is a 2-dimensional time series, which is related to each other. So, either `T_subseq_isconstant` is a 2D bool array, a list of callable functions, a list of 1D bool and callable objects, or a single callable function. Does that answer your question?

Reply:
Yeah! It does! Thank you for the suggestion. I will revise the code accordingly.